### ISSUE TITLE
Allow bandwidth extension for existing VM flavors

### ISSUE THREAD

**Issue by cambierr** [ovh/public-cloud-roadmap#66]
**As a** DevOps,
**I want to** be able to get more bandwidth (paid) on existing VMs and flavors
**So that** I can size my VMs without wasting too many CPUs/RAM just to get enough bandwidth

---

## Description:
Something that I really find annoying on today's public cloud offer is that the bandwidth is fixed per flavor.

While I totally agree on the "default" limit per flavor, I really don't see any valid reason to prevent "buying" more bandwidth per instance/ip/port like we do in the bare metal offers.

**Comment by mhurtrel** [comment]
@cambierr To provide guarranted ressources for all customers at very aggressive price, we tend to prefer to keep our catalog simple and avoid having options/specific scheduling constraints if we can. 

More generally we plan to have more generous networking capabilities on all VM (at equivalent CPU/RAM/Disk ressources) in the next evolution of our instance catalog.

Do you have one or a couple of specific sizing in mind to check if this simple approach would fit ?

For example "i would need someting like a B2-7 but with 1GBPs guaranteed bandwith"

**Comment by cambierr** [comment]
Indeed; for instance we use many B2-15 for site-2-site VPNs and internet gateways for our vracks. Being able to get 1Gbps on them on the public interface would, I think, bring a lot of value since this would prevent us from having to use larger flavors for something we could event handle with a s1-4 in terms of CPU/RAM.

From an economic perspective, I guess the difference between a b2-15 + bandwidth expansion or a base b2-60 wouldn't change a lot but nerdy as I am, I kinda feel stupid wasting that many cores & Gigs of RAM :)

**Comment by BriacBl** [comment]
Hello,
Increasing bandwidth as a separately priced option on VMs isn’t part of our plans, as we aim to keep our catalog straightforward. Our approach is to offer more generous networking capabilities across all VM types. For example, a b3-16 currently provides 1 Gbps, compared to 250 Mbps on the b2-15, while offering the same CPU, RAM, and disk.
